packages/inter-protocol/src/stakeFactory/stakeFactoryManager.js

```diff
@@ -120,7 +120,7 @@ const finish = ({ state, facets }) => {
       helper
         .chargeAllVaults(updateTime)
         .catch(e =>
-          console.error('🚨 runStakeManager failed to charge interest', e),
+          console.error('🚨 stakeFactoryManager failed to charge interest', e),
```

Comment:
Since [unhandled rejections now have good support in the platform](https://github.com/endojs/endo/blob/master/packages/ses/docs/lockdown.md#unhandledrejectiontrapping-options), perhaps it's best to get rid of this catch-and-log?

Reply:
Good thought. In this case, it's intentional to let it slide and carry on so interest charging it attempted again.